CRUDFilters/views.py

```diff
@@ -201,14 +201,16 @@ def _enforce_role_access_control(self):
                 logger.exception("Operation {} cannot be performed on requested object".format(self.request.crud_operation))
                 raise CRUDException("Cannot perform this operation on this object.", status_code=403)
 
+    def process_request(self, request):
+        if request.method == 'OPTIONS':
+            return HttpResponse("Method 'OPTIONS' is not implemented.", status=405)
+
     # For the time being, this only works with token and basic auth.
     def process_view(self, request, view_func, view_args, view_kwargs):
         """
         Perform simple authentication, then check that this user can use this role
         to perform this action (on this item).
         """
-        if not hasattr(view_func, 'cls'):
-            return None
```

Comment:
I presume this is for view functions (as opposed to class-based views) -- why was it removed?